docs/protocol.md

```diff
@@ -750,6 +956,21 @@ See the full specification
                     },
                     "minItems": 0,
                     "$comment": "Return value for a `catchup` request"
+                },
+                {
+                    "$ref": "result/messages_by_channel.json",
+                    "$comment": "Return value for a `get_messages_by_id` request"
+                },
+                {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "serverId": {
+                            "description": "[Base64String] server public key",
+                            "type": "string",
+                            "contentEncoding": "base64"
+                        }
+                    }
```

Comment:
why do you need this? 🤔

Reply:
We are not sure anymore where this came from in `answer.json` but it is removed now 😄